### PR TITLE
calculated the average collaboration team size broken down by. gender

### PR DESCRIPTION
**Pull request by joshyam-k**
here a collaboration team means a group of people who write a paper together

### CODE REVIEW

File: pubs/collab-patterns/collab.Rmd [hunk 1]
```diff
@@ -197,4 +198,66 @@ nonsys_authors %>%
 ```
 
 
+```{r collaborator df creation, eval=FALSE}
```
Comment (eitanf) [inline pubs/collab-patterns/collab.Rmd:201]: You can move this to the setup block

Comment (joshyam-k) [reply pubs/collab-patterns/collab.Rmd:201]: but we don't want this to be evaluated every time, and the setup block does get evaluated each time

**Comment by eitanf** [comment]
I think it looks fine, but this commit history has too many corrections. Can you start fresh with a new pull request and a single commit for this single (?) new feature?

**Comment by joshyam-k** [comment]
this pull request also has quite a few other tables, should I include those in a new pull request too?

**Comment by eitanf** [comment]
Sure, as separate commits if you can.
    On Thursday, August 12, 2021, 03:11:41 PM PDT, Joshua Yamamoto ***@***.***> wrote:  
 
 


this pull request also has quite a few other tables, should I include those in a new pull request too?

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Pull request closed without merging**